notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>print(prediction_response)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
Changed verbatim from `prediction_response` to `print(prediction_response` as suggested.